modules/friendlytag.js

```diff
@@ -207,6 +207,16 @@ Twinkle.tag.callback = function friendlytagCallback() {
 
 		}
 
+		// Add status text element after Submit button
```

Comment:
This is incredibly silly perhaps, but the term `statusElement` is used heavily as `pageobj.getStatusElement()` — frequently shortened as `var statelem` — so, for clarity's sake, I think it'd be better to come up with a different term here.  I don't think "status" quite captures the essence of what this is displaying, but even moving to `statusText` would be clearer I think.  Something like `Twinkle.tag.changes` and `var changesNode` would be clearer as well.  I'm not exactly full of good ideas at the moment, but it's just additional clarity for maintenance down the line, so it need not be beautiful prose.

Reply:
renamed to `statusNode` to avoid confusion with morebits-based `statusElement`s. I agree that status is not the correct word, but then, what is ...

Reply:
Perfect, thanks!